src/utils/cairoTypeSystem.ts

```diff
@@ -169,6 +169,24 @@ export class CairoFelt extends CairoType {
   }
 }
 
+export class CairoUint extends CairoType {
```

Comment:
It adresses #849 so it should be done as one of the first passes. We can merge that as a quick fix for cairo 1.0 but we need a long term solution so please add your comments to #849 @rjnrohit @cicr99 @rodrigo-pino @AlejandroLabourdette